pkg/lightning/backend/backend.go

```diff
@@ -197,6 +198,11 @@ type AbstractBackend interface {
 
 	// LocalWriter obtains a thread-local EngineWriter for writing rows into the given engine.
 	LocalWriter(ctx context.Context, cfg *LocalWriterConfig, engineUUID uuid.UUID) (EngineWriter, error)
+
+	// CollectLocalDuplicateRows collect duplicate keys from remote TiKV storage.
+	CollectLocalDuplicateRows(ctx context.Context, tbl table.Table, sqlMode mysql.SQLMode) error
+
+	CollectRemoteDuplicateRows(ctx context.Context, tbl table.Table, sqlMode mysql.SQLMode) error
```

Comment:
Can add more comments about what `CollectRemoteDuplicateRows` and `CollectRemoteDuplicateRows` do respectively?

Reply:
OK